lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,76 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    def request_params
+        params = super
+        params['search'] ||= 'time >= "1 month ago"'
+    end
+
+    module PerPageAuditLimit
+
+      def self.included(base)
+        base.extend_help do |h|
+          h.section(_('Override audit record pagination')) do
+            h.text(_('Audit records for past one month are returned by default. This can be paginated further by using --per-page and --page options.'))
```

Comment:
This is very nice example of using extended help! Thanks for that!
I think that the text needs some update though, because pagination options won't help you with listing more audits. It's passing `--search` that does the trick.

Reply:
ah! thanks for pointing that out :)

Reply:
How does this text look like? `Audit records for past one month are returned by default. This behavior can be changed by passing --search'`